### PR TITLE
mask-flanneld2

### PR DESCRIPTION
**Pull request by calvix**
rewrite of https://github.com/giantswarm/k8scloudconfig/pull/200

### CODE REVIEW

File: v_0_1_0/worker_template.go [hunk 1]
```diff
@@ -168,13 +168,23 @@ coreos:
     command: stop
     mask: true
   - name: locksmithd.service
+    enable: false
```
Comment (xh3b4sd) [inline v_0_1_0/worker_template.go]: Looks like this change was not part of the closed PR. Is this on purpose?

**Comment by calvix** [comment]
yes

File: v_0_1_0/worker_template.go (hunk 1 above)
Comment (calvix) [reply v_0_1_0/worker_template.go]: yes

File: v_0_1_0/worker_template.go [hunk 2]
```diff
@@ -168,13 +168,23 @@ coreos:
...
     mask: true
   - name: etcd2.service
     enable: false
     command: stop
     mask: true
   - name: fleet.service
+    enable: false
+    command: stop
+    mask: true
+  - name: fleet.socker
```
Comment (r7vme) [inline v_0_1_0/worker_template.go]: typo

**Comment by calvix** [comment]
@r7vme 
Coreos by default is running flanneld in rkt
```
core@ip-172-19-34-202 ~ $ systemctl status flanneld
● flanneld.service - flannel - Network fabric for containers (System Application Container)
   Loaded: loaded (/usr/lib/systemd/system/flanneld.service; disabled; vendor preset: disabled)
   Active: inactive (dead)
     Docs: https://github.com/coreos/flannel
core@ip-172-19-34-202 ~ $ 
core@ip-172-19-34-202 ~ $ cat /usr/lib/systemd/system/flanneld.service
[Unit]
Description=flannel - Network fabric for containers (System Application Container)
Documentation=https://github.com/coreos/flannel
After=etcd.service etcd2.service etcd-member.service
Requires=flannel-docker-opts.service

[Service]
Type=notify
Restart=always
RestartSec=10s
TimeoutStartSec=300
LimitNOFILE=40000
LimitNPROC=1048576

Environment="FLANNEL_IMAGE_TAG=v0.7.1"
Environment="FLANNEL_OPTS=--ip-masq=true"
Environment="RKT_RUN_ARGS=--uuid-file-save=/var/lib/coreos/flannel-wrapper.uuid"
EnvironmentFile=-/run/flannel/options.env

ExecStartPre=/sbin/modprobe ip_tables
ExecStartPre=/usr/bin/mkdir --parents /var/lib/coreos /run/flannel
ExecStartPre=-/usr/bin/rkt rm --uuid-file=/var/lib/coreos/flannel-wrapper.uuid
ExecStart=/usr/lib/coreos/flannel-wrapper $FLANNEL_OPTS
ExecStop=-/usr/bin/rkt stop --uuid-file=/var/lib/coreos/flannel-wrapper.uuid

[Install]
WantedBy=multi-user.target
```

**Pull request merged**